### PR TITLE
Update docs melpa package

### PR DESCRIPTION
**Pull request by mstemm**
Update README to note it's available via MELPA, and fix one minor thing in elisp docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,29 @@ A custom checker for Falco rules files that can be loaded using the [Flycheck](h
 
 ## Setup
 
+### Prerequsites
+
+First, [Install](https://www.flycheck.org/en/latest/user/installation.html) and [enable](https://www.flycheck.org/en/latest/user/quickstart.html) Flycheck if not already installed.
```
Comment (ewilderj) [inline README.md:11]:
```suggestion
First, [install](https://www.flycheck.org/en/latest/user/installation.html) and [enable](https://www.flycheck.org/en/latest/user/quickstart.html) Flycheck if not already installed.
```

File: flycheck-falco-rules.el [hunk 2]
```diff
@@ -71,7 +71,7 @@ flycheck-falco-rules-parse.  ERROR_OR_WARNING_OBJ is from
     "Parse Falco Rules errors/warnings from Falco's validate json output.
 CHECKER and BUFFER denote the CHECKER that returned OUTPUT and
 the BUFFER that was checked respectively.
-See URL `https://github.com/falcosecurity/falco' for more information
+See URL <https://github.com/falcosecurity/falco> for more information
```
Comment (ewilderj) [inline flycheck-falco-rules.el:74]: not falco.org?

File: README.md (hunk 1 above)
Comment (mstemm) [reply README.md]: Good catch, made this change.

**Review by ewilderj — APPROVED** [review]
LGTM

File: flycheck-falco-rules.el (hunk 2 above)
Comment (mstemm) [reply flycheck-falco-rules.el]: Good idea, changed to falco.org.

**Pull request merged**